app/lib/presenter/comment.rb

```diff
@@ -0,0 +1,33 @@
+module Presenter
+  class Comment
+    attr_reader :submission_answers
+
+    def initialize(form_builder_payload:)
+      @submission_answers = form_builder_payload.fetch(:submissionAnswers)
+    end
+
+    REQUEST_METHOD = 'Online form'.freeze
+
+    def optics_payload
+      {
+        Type: type,
+        RequestDate: request_date,
+        RequestMethod: REQUEST_METHOD,
+        AssignedTeam: submission_answers.fetch(:contact_location, ''),
+        'Case.ServiceTeam': submission_answers.fetch(:contact_location, ''),
+        Details: submission_answers.fetch(:feedback_details, '')
+      }
+    end
+
+    private
+
+    def type
+      ENV['COMMENT_TYPE'] || ''
+    end
+
+    def request_date
```

Comment:
since this is the same method as the one in the complaints we can move this to a module and include it in each presenter that requires it